src/async_std.rs

```diff
@@ -5,19 +5,18 @@
 //!   class="module-item stab portability"
 //!   style="display: inline; border-radius: 3px; padding: 2px; font-size: 80%; line-height: 1.2;"
 //! ><code>unstable-streams</code></span>
-//! are only available when the `unstable-streams` Cargo feature is enabled:
+//! >are only available when the `unstable-streams` Cargo feature is enabled:
```

Comment:
I think this is incorrect, the `>` on the line before closes the `<span>` tag
```suggestion
//! are only available when the `unstable-streams` Cargo feature is enabled:
```